CHANGELOG.md

```diff
@@ -1,4 +1,22 @@
-# Changelog
+## (Unreleased)
```

Comment:
I understand this will be updated automatically in master once this project is in place

Reply:
correct, left a note in the PR description